reference/api/index.rst

```diff
@@ -0,0 +1,180 @@
+Web API
+=======
+
+.. toctree::
+
+    rest
+    jobs
+
+How it's structured
+-------------------
+
+There is actually two APIs. A REST API with basic CRUD operations, and a jobs API which can launch heavy processes asynchronously.
+The current version of the API is v1. All URI have to request explicitly this version.
+
+.. code-block:: bash
+
+    GET https://demo.akeneo.com/api/rest/v1
+    GET https://demo.akeneo.com/api/jobs/v1
+
+Request these URIs to get all the endpoint categories that the API supports.
```

Comment:
"Request these URIs" => By requesting these two URI, you will get the list of all available endpoints.

You can add as well an excerpt of the response to give an idea to the users.